sdk/provisioning/Azure.Provisioning.CloudMachine/src/AzureSdkExtensions/OpenAIFeature.cs

```diff
@@ -8,21 +8,30 @@
 using Azure.Provisioning.Authorization;
 using Azure.Provisioning.CognitiveServices;
 using OpenAI.Chat;
+using OpenAI.Embeddings;
 
 namespace Azure.Provisioning.CloudMachine.OpenAI;
 
 public class OpenAIFeature : CloudMachineFeature
 {
-    public string Model { get; }
-    public string ModelVersion { get; }
+    private OpenAiModelDeployment _chatDeployment;
+    private OpenAiModelDeployment? _embeddingsDeployment;
 
-    public OpenAIFeature(string model, string modelVersion) {  Model = model; ModelVersion = modelVersion; }
+    public OpenAIFeature(OpenAiModelDeployment chatDeployment, OpenAiModelDeployment? embeddingsDeployment = default)
+    {
+        if (chatDeployment == null)
+        {
+            throw new ArgumentNullException(nameof(chatDeployment));
+        }
+        _chatDeployment = chatDeployment;
+        _embeddingsDeployment = embeddingsDeployment;
+    }
 
     public override void AddTo(CloudMachineInfrastructure cloudMachine)
     {
         CognitiveServicesAccount cognitiveServices = new("openai")
         {
-            Name = cloudMachine.Id,
+            Name = $"{cloudMachine.Id}/{cloudMachine.Id}",
```

Comment:
why this change?

Reply:
I noticed that this is how AI Studio names them when it deploys the models.